tests/unit/amo/pages/TestAddon.js

```diff
@@ -1527,6 +1518,19 @@ describe(__filename, () => {
       expect(currentVersion).toEqual(createInternalVersion(apiVersion));
     });
 
+    it('sets the version to null for an add-on without a version', () => {
+      signIn();
+      fetchAddon({
+        addon: {
+          ...fakeAddon,
+          current_version: null,
+        },
+      });
+
+      const { currentVersion } = _mapStateToProps();
+      expect(currentVersion).toEqual(null);
+    });
```

Comment:
It'd be better to not test `_mapStateToProps()`, so I'd like to see something like:

```js
  it('renders without a version', () => {
    const { store } = dispatchSignInActions();
    const addon = { ...fakeAddon, current_version: null };

    store.dispatch(_loadAddonResults({ addon }));

    const root = renderComponent({ store });

    expect(root.find('.Addon')).toHaveLength(1);
    expect(root.find('.Addon')).toHaveProp('data-site-identifier', addon.id);
  });
```

Reply:
This test will fail if you revert your fix.